Source/Synthesis.Solr/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Reflection;
```

Comment:
This should copy the properties model of other Synthesis assemblies (e.g. https://github.com/kamsar/Synthesis/blob/master/Source/Synthesis/Properties/AssemblyInfo.cs) and adding a _link_ (https://msdn.microsoft.com/en-us/library/windows/apps/jj714082(v=vs.105).aspx) to the SharedAssemblyInfo.cs in the root of source so that all assemblies share version info.